pom.xml

```diff
@@ -1341,6 +1341,9 @@
                         <signaturesFile>${project.parent.basedir}/codestyle/joda-time-forbidden-apis.txt</signaturesFile>
                         <signaturesFile>${project.parent.basedir}/codestyle/druid-forbidden-apis.txt</signaturesFile>
                     </signaturesFiles>
+                    <excludes>
+                      <exclude>**/SomeAvroDatum.class</exclude>
```

Comment:
I've created https://issues.apache.org/jira/browse/AVRO-2731 and https://github.com/apache/druid/issues/9331 to track this. FYI @Fokko.

@capistrant please try to create such issues yourself to close the loops and offload info from the heads of people.